### ISSUE TITLE
Indonesia GTM - Team 7: Launch Event

### ISSUE THREAD

**Issue by RizRiyz** [mozilla/webmaker-launch#50]
Goal:
- Organize launch on the first week of August (date TBD)
- Press conference should be held in weekdays (estimated 11 - 14 August)

Team Member: @rara79, @andi-belutz, @alamanda, @pixelglass, @ariestiyansyah, @williamq, Rosana Ardila 
Team Pilot: @rara79

Plan: https://docs.google.com/document/d/1tVJZfGiwAD7SGCskzi4NbJvaKzRVSqZ18nR5s8HiuTo/edit?usp=sharing

**Comment by alamanda** [comment]
@secretrobotron 
![pricelist-swag-indonesia](https://cloud.githubusercontent.com/assets/12945788/8267681/49d219e6-1795-11e5-896f-fd0f40f32f21.jpg)

**Comment by alamanda** [comment]
- Ina found a supplier who can speak english.
- t-shirt will be produced and ready on july 30th for launching in august and produced locally. 
- quotation will be ready on monday (june 29)

**Comment by secretrobotron** [comment]
Figuring out the format:
- 1 launch with 300+ people in Jakarta
- 5 launches with fewer people in different cities?

**Comment by andi-belutz** [comment]
Need to be discuss

**For launch event:**
- Launch event concept
- Target audience (eg. how many reporters, students, teachers, bloggers)
- Venue
- Mozilla person(s) attending

**Post Launch event:**
- Launch event in other cities(?)
- Schools Roadshow
- Webmaker contest(s)

**Comment by rara79** [comment]
- @alamanda  has followed up more with Karen. The proforma invoice for 50% down payment has been sent. Waiting for the payment right away. 
- Need to be careful with swag allocation for the events scheduled in August 1-4th, as everything will be ready by August 4th. The printer had a sudden and extensive  order last week when @alamanda  was out in Scotland without connection!:( 
- @rara79 and @williamq met on Monday - plan is to organize small events in 5 cities across Indonesia.

Attendees
- Jakarta : 200-300 attendees
- Bandung, Semarang, Surabaya, Makassar : 100 attendees each cities.

Target audiences:
- Mozillians (Reps/FSA/etc)
- Social media influencers 
- Top bloggers
- Media  (online / offline)
- Partner or potential partners? 

Offline meetup. @andi-belutz 's office.
8 pm or 9 pm. Monday, July 13th, 2015.

**Comment by alamanda** [comment]
had a quick chat with @rara79 just now. 
I think the venue we'll use for launching in Jakarta should be a place that's ready to use. I dig for HongKong Cafe because transport wise is very easy to reach using all public transport mode available in Jakarta. The location is very central (15 steps away from Artotel, which most likely will be the hotel where William's going to stay) in Central Jakarta. Good food selection with good price (not too expensive), so we don't have to source for catering outside. 
And I think, it will be easy to set up extra internet connection if necessary in this place. Am I right @andi-belutz?

**Comment by secretrobotron** [comment]
Some links for events from @rara79:
- http://www.eventbrite.com/e/mozbelajar-webmaker-app-for-android-tickets-17965893471
- https://reps.mozilla.org/e/mozbelajar-jakarta-webmaker-app-for-android/

Exciting!

cc @HPaulJohnson @KevZawacki @williamq

**Comment by rara79** [comment]
Workshops in 5 different cities in Indonesia, starting August 15th, 2015.

Jakarta: August 15th, 2015
https://reps.mozilla.org/e/mozbelajar-jakarta-webmaker-app-for-android/

Bandung: August 15th, 2015
https://reps.mozilla.org/e/mozbelajar-bandung-webmaker-app-for-android/

Semarang: August 15th, 2015
https://reps.mozilla.org/e/mozbelajar-semarang-webmaker-app-for-android/

Surabaya: August 30th, 2015
https://reps.mozilla.org/e/mozbelajar-surabaya-webmaker-app-for-android/

Makassar: August 29th, 2015
https://reps.mozilla.org/e/mozbelajar-makassar-webmaker-app-for-android/

**Comment by alamanda** [comment]
@secretrobotron we've been looking for you since yesterday:) did you receive any of my messages in telegram?

**Comment by alamanda** [comment]
@williamq @secretrobotron please find the **fireladies** plan for surabaya in this link: https://etherpad.mozilla.org/school-visit-sub

**Comment by rara79** [comment]
@secretrobotron @williamq here's the new teaching kit for Webmaker App, using the awesome new thimble, remixed from bobby's teaching kit and my old teaching kit. Check this out: https://d157rqmxrxj6ey.cloudfront.net/rara/4349/

**Comment by alamanda** [comment]
@rara79 how is our status in sending releases to the press? It's -7days to the day. We should make sure that many journalists show up to our event.

**Comment by KevZawacki** [comment]
@alamanda @rara79 If I can help at all with press releases / etc., please let me know! 
Also, FYI, we are still waiting to hear back from XM Gravity and Juara Agency.

**Comment by andi-belutz** [comment]
@KevZawacki who do you contact in XMGravity?

**Comment by KevZawacki** [comment]
@andi-belutz I emailed their general mailbox (info@xmgravity.com ), but haven't heard back. Do you have a specific contact I can reach out to?

**Comment by andi-belutz** [comment]
@KevZawacki I will ask around, will have the answer for you by tomorrow

**Comment by rara79** [comment]
@KevZawacki same question with andi but different subject, who do you contact in Juara Agency?

**Comment by KevZawacki** [comment]
Hi @rara79, I contacted them through [the form on their website](http://juara-agency.com/contact/), and also emailed admin@juara-agency.com. Do you have any contacts there?

**Comment by rara79** [comment]
@KevZawacki you can contact Wiwik for Juara, her emails are wiwik@juara-agency.com and wiwik.wae@gmail.com.

**Comment by andi-belutz** [comment]
@KevZawacki for XM Gravity you can email Dimas Novriandi at dimas.novriandi@mirumagency.com and cc the email to Deden Purnamahadi at deden.purnamahadi@mirumagency.com
For addition, you can also contact Provetic (www.provetic.com) at nunik99@provetic.com

**Comment by KevZawacki** [comment]
Thank you, @rara79 and @andi-belutz. I'm emailing now, and will update this ticket when I hear back.

**Comment by andi-belutz** [comment]
@KevZawacki please let us know if they still don't answer your email. Btw, tomorrow is public holiday in Indonesia, so maybe you will get answer from them by Friday. But for XM and Provetic, I already give them a heads up. So if you done emailing them, you can ping me and I can ping them. Also I can help ping Wiwik from Juara, I know her also :)

**Comment by KevZawacki** [comment]
Great! Yes, I've just sent the emails -- OK to ping. :-)

**Comment by andi-belutz** [comment]
@KevZawacki I have ping all of them, 2 have responded to me. While waiting their formal reply to your email, let me recap our informal chat:
- XM Gravity
  Dimas said that what you requested is not their company scope of work, they are more into integrated digital marketing while what you requested is more like a work for PR Agency. So I ask him, what PR Agency that he recommend, and he recommend Burson Masteller and you can email adham.somantrie@bm.com (I already give him a heads up).
- Provetic
  What you requested is part of their scope of work. She will reply your email, and she will discuss it first with her team.

**Comment by KevZawacki** [comment]
Thanks, @andi-belutz!

**Comment by rara79** [comment]
@KevZawacki, I got a word from Wiwik (Juara Agency) that she already got your email and already reply to you, too. That's all :)

**Comment by rara79** [comment]
September 29th event:
Eventbrite URL: https://www.eventbrite.com/e/webmaker-app-party-tickets-18728634851
Reps event URL: https://reps.mozilla.org/e/webmaker-app-party/
@secretrobotron @HPaulJohnson @williamq

**Comment by KevZawacki** [comment]
@rara79 I did! We're going to chat after the holiday :-)

**Comment by KevZawacki** [comment]
Hi all, here's an update on our PR agency outreach:
- Provetic -- available to help. I've shared additional information about our Webmaker work in Indonesia, and am waiting to hear back re: campaign suggestions and rates
- Juara Agency -- closed for the holiday, and will provide campaign suggestions and rate card on Monday
- Mirium -- does not specialize in PR
- Burson-Marsteller -- has Opera as a client, and so cannot work with Mozilla at the moment

Thanks again, @rara79 and @andi-belutz, for your help. I will continue discussions Provetic and Juara. But, it's unlikely they will be able to help with Tuesday's event. 

So: how can I help directly? We have the great media list from August. Can I help to write a pitch? Assist in any other way?

**Comment by rara79** [comment]
@KevZawacki can you help us to make a QA list for us. A list something like Anticipated Q&A from media, what to answer. Also if you can give us a document of facts about Webmaker App. These documents may help us to prepare face the media. 

If you can do it, please send us before Tuesday Sept 29th, Jakarta time :) Thank you!

**Comment by KevZawacki** [comment]
@rara79 Absolutely! I've put together a document, [here](https://docs.google.com/document/d/1_1NCKKERveaw0dkAyek7TugClEHeIBievs0CBF-mf8U/edit), with lots of information and answers. Please let me know if/how I can update it further.

**Comment by rara79** [comment]
@KevZawacki this is perfect! Thank you so much!

**Comment by alamanda** [comment]
Thank you so much for compiling this together, Kevin! It's super helpful:) 
Wish us luck for tonight. I hope many media show up. 

cheers!

**Comment by KevZawacki** [comment]
@rara79 @alamanda Hey, have you guys spotted any press coverage from Tuesday's event?

PS -- Check out the social media results [here](https://docs.google.com/document/d/1LbntletzsdOi6TWw6xqWRKUT1BLu2JWxshCVPAcMnVY/edit). Nice work! :smile:
